src/components/FilterableTable.react.js

```diff
@@ -0,0 +1,45 @@
+import {FilterBarActor} from "../actors/FilterBarActor";
+import {TableActor} from "../actors/TableActor";
+
+import {FilterBarStore} from "../stores/FilterBarStore";
+import {TableStore} from "../stores/TableStore";
+
+import {FilterBar} from "./FilterBar/FilterBar.react";
+import {Table} from "./Table/Table.react";
+
+export class FilterableTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.filterBarStore = new FilterBarStore(props.filterBarConfiguration);
+    this.tableStore = new TableStore(props.tableConfiguration);
+
+    this.filterBarActor = new FilterBarActor(this.filterBarStore, this.tableStore);
+    this.tableActor = new TableActor(this.filterBarStore, this.tableStore);
+  }
+
+  getChildContext() {
```

Comment:
They haven't yet, but I think they've indicated intentions to do so. (https://facebook.github.io/react/blog/2014/03/28/the-road-to-1.0.html#context)[ReactJS site]. They also expose it in a couple of their external calls, namely React.children.

I guess there is a risk in using an as yet undocumented method, but the fix would just be explicitly passing the stores and actors as props down through the tree.